packages/yum/almalinux-8/Dockerfile

```
@@ -10,7 +10,8 @@ RUN \
   dnf install -y ${quiet} \
     epel-release \
     'dnf-command(config-manager)' \
-    https://packages.groonga.org/almalinux/8/groonga-release-latest.noarch.rpm && \
+    https://packages.groonga.org/almalinux/8/groonga-release-latest.noarch.rpm \
+    https://apache.jfrog.io/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm && \
```

packages.apache.org is an alias of apache.jfrog.io but it's better because it uses apache.org domain:

```suggestion
    https://packages.apache.org/artifactory/arrow/almalinux/$(cut -d: -f5 /etc/system-release-cpe | cut -d. -f1)/apache-arrow-release-latest.rpm && \
```